src/main/scala/dsptools/VerboseDspTesterUtils.scala

```diff
@@ -0,0 +1,135 @@
+package dsptools
+
+import chisel3._
+import java.io.{File, FileWriter, BufferedWriter}
+import chisel3.iotesters.TestersCompatibility
+
+// Note: This will dump as long as genVerilogTb is true (even if you're peeking/poking DspReal)
+trait VerilogTbDump {
```

Comment:
Nope. I tried earlier for a different reason. I couldn't figure out how to use it properly. But overriding step/reset serves other purposes [and overriding any functions in general].

1) Verbose = make the logger print more useful
2) Dumps to a TB file. 

I'd rather keep only stuff that dumps to a TB file in the trait.